app.js

```diff
@@ -1,100 +1,79 @@
 import React from 'react';
-import 'highcharts';
-import ReactHighcharts from 'react-highcharts';
+import Chart from './charts/baseChart';
 
 /**
- * Component representing a single chart
+ * Component for displaying the top-level dashboard
  */
-class Chart extends React.Component {
+class Dashboard extends React.Component {
     /**
-     * Create a new chart
-     * @param props props of the chart
+     * @inheritdoc
      */
     constructor(props) {
         super(props);
-        this.state = {
-            data: null,
-            error: null
-        };
-    }
+        this.psnrTestResultTransformFunc = jsonData => {
```

Comment:
this whole thing being defined here doesn't feel great to me. let me know if you have a suggestions